ui/jobs/combo_tracker.ts

```diff
@@ -1,7 +1,19 @@
 import { kAbility, kComboBreakers } from './constants';
 
+type StartMap = Record<string, {
```

Comment:
Same comment as the other patch, but please use an index signature here like `{ [skill: string]: etc }` instead of `Record<string, etc>`

Reply:
Sure!